temporal-sdk/src/main/java/io/temporal/worker/WorkflowImplementationOptions.java

```diff
@@ -73,21 +75,30 @@ public Builder setActivityOptions(Map<String, ActivityOptions> activityOptions)
       return this;
     }
 
+    public Builder setDefaultActivityOptions(ActivityOptions defaultActivityOptions) {
```

Comment:
Add comments to setActivity options and setDefaultActivityOptions which explain their use and precedence.